home-server-generator/src/main/resources/mustaches/server/bootstrap.mustache

```diff
@@ -0,0 +1,14 @@
+package {{apiPackage}};
+
+import io.swagger.models.*;
+
+import io.swagger.models.auth.*;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+
+public class Bootstrap {
```

Comment:
Can't we exclude it from generating instead of adding an empty template?

Reply:
Ok, noted